crates/sncast/tests/data/scripts/script_manual_tests/src/lib.cairo

```diff
@@ -0,0 +1,3 @@
+mod test_deploy;
+mod test_declare_after_invoke;
+mod test_two_undeployed_declare;
```

Comment:
can you share specific scenarios where flows shouldn/shouldn't work? What I mean is, we should think about how to write tests that show us that something goes wrong (if possible on devnet), these all seems like happy cases to me (and most of them are probably already covered in our existing test cases, right?)

Reply:
yep, all of them are heppy cases. No idea what could failure